packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/choiceAt.put..st

```diff
@@ -0,0 +1,13 @@
+accessing
+choiceAt: anIndex put: aBoolean
+|newSelectedChoices|
+	newSelectedChoices := selectedChoices
+		perform: (aBoolean ifTrue: [ #copyWith: ] ifFalse: [ #copyWithout: ])
```

Comment:
TLDR: I personally deem it too resource intensive to come up with our own way of implementing `MultiSelectionLists`, and would rather stick with how Squeak does it.

Simply accessing `selectedChoices` using `at:` is not possible, since `selectedChoices` is a `Set`. Perhaps it would also work as an `OrderedCollection` or any other indexable collection, but we would need to look into that, as all implementations of `MultiSelectionLists` use Sets. And even then, we do not put `aBoolean` at `anIndex` but rather the list entry at `anIndex`.
If you want, we can look at this at a later point, but I personally would rather not, as it was difficult enough to "copy" the existing implementation

Reply:
Makes sense. About `perform:` Could we have two branches instead?

```st
choice := self choiceList at: anIndex ifAbsent: [ ^ self ]
newSelectedChoices := aBoolean ifTrue: [ selectedChoices copyWith: choice ] ifFalse: [ selectedChoices copyWithout: choice ]
```

Reply:
Following my initial testing it seems your approach works as intended. Since it is easier to understand than `perform:with:` I will include it in the commit.